mlrun/config.py

```diff
@@ -288,6 +288,12 @@
                     "pull_state_interval": 5,  # seconds
                 },
             },
+            "nuclio": {
+                # setting interval to a higher interval than regular jobs / build, because pulling the retrieved logs
+                # from nuclio for the deploy status doesn't include the actual live "builder" container logs, but
+                # rather a high level status
+                "pull_deploy_status_default_interval": 10  # seconds
+            },
```

Comment:
Maybe add it to the already existing `nuclio` field in the `httpdb` config?
So all nuclio-related configurations will be in the same place.

Reply:
I actually think its better to have it under the logs struct, you can see that we also have pipeline configuration above and config related to jobs below.